stdpopsim/genomes.py

```diff
@@ -309,21 +312,23 @@ def species_contig(
     def length(self):
         return self.recombination_map.sequence_length
 
-    @property
```

Comment:
I think it's a mistake to make this a @property, since it can be an expensive array to make, and if people can call the method like `contig.all_intervals_array` then they'll do things like:
```
for j in range(1000):
   x = contig.all_intervals_array[j, :]
   # etc
```
which calls this method every time through the loop.